src/index.js

```diff
@@ -27,3 +28,9 @@ require('./brushes/stamp.js');
 require('./brushes/spheres.js');
 require('./brushes/cubes.js');
 require('./brushes/rainbow.js');
+
+var f = function() {
+  requestAnimationFrame(f)
+  AFRAME.TWEEN.update()
+}
+requestAnimationFrame(f)
```

Comment:
This is just to fix the chrome animation frame timing bug, will remove before committing.